mkdocs.yml

```diff
@@ -114,6 +114,7 @@ nav:
         - Permission system: permission_management/permission_system.md
         - Work with permissions: permission_management/work_with_permissions.md
         - Manage users: user_management/manage_users.md
+    - Recent activity: recent_activity/recent_activity.md
```

Comment:
I would move the article to "Permissions and users" because it is about tracking user activity.

Reply:
My first idea what to put it into "Content management" but it wasn't totally fit in. The user part of "Permissions and users" might effectively suit.

Reply:
Moved in b33470afcb36f81ceeb0a6be9db17be4b8aea7e6